### PR TITLE
Radius parameter for /deleteblocks here

### PR DESCRIPTION
**Pull request by SmallJoker**
Adds an additional radius parameter (in nodes) to the command `/deleteblocks here`

### CODE REVIEW

File: builtin/game/chatcommands.lua [hunk 1]
```diff
@@ -404,13 +404,14 @@ core.register_chatcommand("set", {
 })
 
 core.register_chatcommand("deleteblocks", {
-	params = "[here] [<pos1> <pos2>]",
+	params = "[here [radius]] [<pos1> <pos2>]",
```
Comment (ShadowNinja) [inline builtin/game/chatcommands.lua]: Should be `(here [radius])|(<pos1> <pos2>)` or similar.

**Comment by nerzhul** [comment]
hi, thanks for your PR.

please change the syntax like @ShadowNinja said before merge

**Comment by SmallJoker** [comment]
Fine, I updated the syntax description.

**Comment by C1ffisme** [comment]
Question : Why would you use /deleteblocks? I mean, maybe you can use it for updating unused chunks to have ores from the latest mapgen mod, but other than that...

**Comment by SmallJoker** [comment]
You can repair broken areas and plant trees without saplings.

**Comment by C1ffisme** [comment]
Broken? By player?

**Comment by SmallJoker** [comment]
@C1ffisme by player of by server crash / mapgen error

**Comment by C1ffisme** [comment]
@SmallJoker Ah, okay.

File: builtin/game/chatcommands.lua [hunk 2]
```diff
@@ -418,6 +419,12 @@ core.register_chatcommand("deleteblocks", {
 			end
 			p1 = player:getpos()
 			p2 = p1
+
```
Comment (Zeno-) [inline builtin/game/chatcommands.lua]: trailing whitespace

File: builtin/game/chatcommands.lua [hunk 3]
```diff
@@ -418,6 +419,12 @@ core.register_chatcommand("deleteblocks", {
 			end
 			p1 = player:getpos()
 			p2 = p1
+			
+			if #args >= 2 then
+				local radius = tonumber(args[2]) or 0
+				p1 = vector.add(p1, radius)
+				p2 = vector.subtract(p2, radius)
+			end
```
Comment (Zeno-) [inline builtin/game/chatcommands.lua]: trailing whitespace

**Comment by Zeno-** [comment]
Can everyone please be aware of whitespace errors, please?

I've fixed them in this PR before merging but it would be nice to see PRs without them.

**Comment by Zeno-** [comment]
commit bbed01ab62bf2433e103dd18c6c07ff3f503bc57

**Pull request closed without merging**